### PR TITLE
@tc/preview changes existing docs

### PR DESCRIPTION
**Pull request by tcdavis**
# Why

Updates existing docs pages referencing development clients to point to our new pages

# How

`ag Development Client` 
Update pages

# Test Plan

Ran locally and followed all links

### CODE REVIEW

**Comment by tcdavis** [comment]
partially addresses ENG-1422

File: docs/pages/workflow/customizing.md [hunk 1]
```diff
@@ -37,12 +37,12 @@ If you want to make static changes to your native project files like the iOS `In
 
 If you've decided that you want to roll your app back to being fully managed (no iOS and Android projects in your project directory), you can checkout your most recent commit before executing `expo run:[ios|android]`, then run `npm install` again to restore the state of your `node_modules` directory.
 
-## Releasing apps with custom native code to production
+## Developing apps with custom native code
 
-The classic `expo build` command does not support custom native code. When you're ready to ship your app, you can [build it with EAS Build](/build/introduction.md) or archive and sign it locally.
+Once you have customized the native code in their project, you can use the [`expo-dev-client`](/clients/introduction.md) package to create a custom development client and retain the convenience of working with just JavaScript and/or TypeScript in Expo Go. You can create a custom client for your managed or bare workflow by following [our guide](/clients/getting-started.md).
```
Comment (cruzach) [inline docs/pages/workflow/customizing.md:42]:
```suggestion
Once you have customized the native code in your project (by installing a new package outside the SDK, or by writing your own), you can use the [`expo-dev-client`](/clients/introduction.md) package to create a custom development client and retain the convenience of working with just JavaScript and/or TypeScript in Expo Go. You can create a custom client for your managed or bare workflow by following [our guide](/clients/getting-started.md).
```

**Review by cruzach — APPROVED** [review]

File: docs/pages/workflow/customizing.md (hunk 1 above)
Comment (tcdavis) [reply docs/pages/workflow/customizing.md:42]: 🙏  I think the rest of the page should be providing the context around how to customize the code so we don't need the callout.
Fixing the "their" though!

File: docs/pages/workflow/customizing.md [hunk 2]
```diff
@@ -37,12 +37,12 @@ If you want to make static changes to your native project files like the iOS `In
...
+## Developing apps with custom native code
 
-The classic `expo build` command does not support custom native code. When you're ready to ship your app, you can [build it with EAS Build](/build/introduction.md) or archive and sign it locally.
+Once you have customized the native code in your project, you can use the [`expo-dev-client`](/clients/introduction.md) package to create a custom development client and retain the convenience of working with just JavaScript and/or TypeScript in Expo Go. You can create a custom client for your managed or bare workflow by following [our guide](/clients/getting-started.md).
 
-<TerminalBlock cmd={['# Install the CLI', 'npm i -g eas-cli', '', '# Build your app!', 'eas build -p all']} />
+## Releasing apps with custom native code to production
 
-## Coming soon: "Expo Development Client"
+The classic `expo build` command does not support custom native code. When you're ready to ship your app, you can [build it with EAS Build](/build/introduction.md) or archive and sign it locally.
```
Comment (brentvatne) [inline docs/pages/workflow/customizing.md:46]:
```suggestion
The classic build system (`expo build`) does not support custom native code. When you're ready to ship your app, you can [build it with EAS Build](/build/introduction.md) or archive and sign it locally.
```

**Pull request merged**